wg-reliability/sippy/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sippy
+  name: sippy
+  namespace: wg-reliability-sippy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sippy
+  template:
+    metadata:
+      labels:
+        app: sippy
+    spec:
+      containers:
+        - image: quay.io/bparees/sippy:kube-latest
+          imagePullPolicy: Always
+          name: sippy
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          resources:
+            limits:
+              memory: 1500M
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          command:
+            - /tmp/src/sippy
+          args:
+            - --server
+            - --end-day
+            - "7"
+            - --local-data
+            - /data
+            - --dashboard
+            - kube-master=sig-release-master-blocking,sig-release-master-informing=
```

Comment:
Do we want both? Or maybe we want to start just with blocking?

Or actually, I guess it's probably moot anyway, because we probably want to create our own dashboard for this purpose...